### PR TITLE
Added mappings for some new MITPE topics

### LINKED ISSUE #3178: KeyError: 'Digital Transformation'
Sentry Issue: [DISCUSSIONS-2NA](https://sentry.io/organizations/mit-office-of-digital-learning/issues/1617244699/?referrer=github_integration)

```
KeyError: 'Digital Transformation'
  File "course_catalog/etl/utils.py", line 223, in map_topics
    for topic in mapping[strip_extra_whitespace(raw_topic)]:

No topic mapping found for  Digital Transformation in {"Biotechnology & Pharmaceutical": ["Biological Engineering"], "Computer Science": ["Computer Science"], "Crisis Management": ["Management"], "Data Modeling & Analytics": ["Computer Science"], "Design & Manufacturing": ["Systems Engineering"], "Energy & Sustainability": ["Energy", "Earth Science"], "Imaging": ["Computer Science", "Electrical Engineering"], "Innovation": ["Innovation"], "Leadership & Communication": ["Leadership", "Communication"], "Radar": ["Electrical Engineering"], "Real Estate": ["Rea...
```

### PR DESCRIPTION
**Pull request by rhysyngsun**
@pdpinch the mappings we discussed

#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Fixes #3178 

#### What's this PR do?
Maps a couple new topics that showed up

#### How should this be manually tested?
Run `./manage.py backpopulate_mitpe_data` and verify courses that have these topics on https://professional.mit.edu/course-catalog are correctly mapped.

### CODE REVIEW

**Review by abeglova — APPROVED** [review]
👍

File: course_catalog/constants.py [hunk 1]
```diff
@@ -116,11 +116,13 @@ class PrivacyLevel(Enum):
     "Computer Science": ["Computer Science"],
     "Crisis Management": ["Management"],
     "Data Modeling & Analytics": ["Computer Science"],
+    "Digital Transformation": ["Business", "Information Technology"],
```
Comment (pdpinch) [inline course_catalog/constants.py:119]:
```suggestion
    "Digital Transformation": ["Information Technology"],
```
"Business" is too broad a match for this. I think just "Information Technology" is OK.

File: course_catalog/constants.py [hunk 2]
```diff
@@ -116,11 +116,13 @@ class PrivacyLevel(Enum):
...
     "Design & Manufacturing": ["Systems Engineering"],
     "Energy & Sustainability": ["Energy", "Earth Science"],
     "Imaging": ["Computer Science", "Electrical Engineering"],
     "Innovation": ["Innovation"],
     "Leadership & Communication": ["Leadership", "Communication"],
+    "Legal Tech": ["Social Science", "Legal Studies"],
```
Comment (pdpinch) [inline course_catalog/constants.py:125]:
```suggestion
    "Legal Tech": ["Information Technology", "Legal Studies"],
```

Based on the courses, I don't think these are a broad match for "Social Science" either. But I do think ""nformation Technology" is a fair match.

**Review by pdpinch — CHANGES_REQUESTED** [review]
Sorry, I refined my thinking a bit since last night.

**Review by pdpinch — APPROVED** [review]

**Pull request merged**